getting_started/first_3d_game/06.jump_and_squash.rst

```diff
@@ -275,30 +280,30 @@ With this code, if no collisions occurred on a given frame, the loop won't run.
 
  That's a lot of new functions. Here's some more information about them.
 
-The functions ``get_slide_count()`` and ``get_slide_collision()`` both come from
-the :ref:`CharacterBody3D<class_CharacterBody3D>` class and are related to
+The functions ``get_slide_collision_count()`` and ``get_slide_collision()`` both come from
+the :ref:`CharacterBody3D <class_CharacterBody3D>` class and are related to
 ``move_and_slide()``.
 
 ``get_slide_collision()`` returns a
 :ref:`KinematicCollision3D<class_KinematicCollision3D>` object that holds
 information about where and how the collision occurred. For example, we use its
-``collider`` property to check if we collided with a "mob" by calling
-``is_in_group()`` on it: ``collision.collider.is_in_group("mob")``.
+``get_collider`` property to check if we collided with a "mob" by calling
+``is_in_group()`` on it: ``collision.get_collider(index).is_in_group("mob")``.
 
 .. note::
 
     The method ``is_in_group()`` is available on every :ref:`Node<class_Node>`.
 
 To check that we are landing on the monster, we use the vector dot product:
-``Vector3.UP.dot(collision.normal) > 0.1``. The collision normal is a 3D vector
+``Vector3.UP.dot(collision.get_normal) > 0.1``. The collision normal is a 3D vector
```

Comment:
Does this work?

Reply:
yup, just confirmed it again in Godot beta10

Reply:
Nope, this is still a typo. It should be `Vector3.UP.dot(collision.get_normal())`. The code in the examples is correct, but here it is wrong. You probably tested the examples, which don't have a typo.

Reply:
You are right, I missed the `()` and as you said, I tested the example and saw `get_normal` working so I wrote the above lol

Anyway, fixed in latest push :+1: